controllers/update_config_map.go

```diff
@@ -81,20 +84,39 @@ func (u UpdateConfigMap) Reconcile(r *FoundationDBClusterReconciler, context ctx
 		return false, err
 	}
 
+	var errs []error
+	// We try to update all instances and if we observe an error we add it to the error list.
+	// TODO: should we try to update the instances concurrently?
```

Comment:
We could also try to update all instances "at once" since we probably waiting for I/O the most time.